lib/rules/layer-name-pattern/README.md

```diff
@@ -0,0 +1,85 @@
+# layer-name-pattern
+
+Specify a pattern for layer names.
+
+<!-- prettier-ignore -->
+```css
+@layer foo {}
+/**    ↑
+ * This layer name */
+```
+
+<!-- prettier-ignore -->
+```css
+@import "foo.css" layer(bar);
+/**                     ↑
+ *  And this layer name */
+```
+
```

Comment:
```suggestion
```

We typically have a single [prototypical code example](https://stylelint.io/developer-guide/rules#write-the-readme) at the top.